README.md

```diff
@@ -104,6 +104,10 @@ Windows 7 and later are supported
 
 [More information](https://github.com/electron/electron/blob/master/docs/tutorial/supported-platforms.md).
 
+### App Start-Options
```

Comment:
I think it would be better if we could add some kind of explanation why it's here. Also, I guess it's a little too short to has its own section. Maybe move it under Development?

Reply:
> I think it would be better if we could add some kind of explanation for why it's here and what it does. Or better yet, reference the issue(s) that led to this change. I think Linux users will appreciate it.

Yes, thats a good idea. I make a proposal.

> Also, it seems a little too short to has its own section, maybe move it under Development?
Is it really related to development? I also thought about adding a start option for a path to a custom theme, so maybe the section gets bigger in the future :) If it shouldn't be part of the readme, maybe it should get an own wiki page?

Reply:
Yes, I think it's related to Development because you'll only use those options for development purpose. But I think it deserves its own Wiki page as well. Maybe we can title the page as "Debug in Production" or "Debug Exported App"?

I just updated the Wiki setting so everyone can contribute now, and I think the page should include this: 

> Dev-Server: electron . --force-devtools
> Linux AppImage: ./Manta-1.1.2-x86_64.AppImage --force-devtools
> Installed App (.deb): manta --force-devtools

Reply:
Yes, the (1) "--force-devtools" option is really related to development, but (2) "--disable-hardware-acceleration" is a user setting for the app. So we should split the documentation of these two params to 2 different places. For (1) a "Debug Exported App" wiki page would fit. I would leave infos about (2) in the main readme.

Reply:
> I think it would be better if we could add some kind of explanation for why it's here and what it does.

I don't know what to explain. The title of the section and the paramname explain everything there is to explain, or?
Maybe: To give the user the possiblity to easily set some options for the app ...

Reply:
I just added a short description to the README. Please have a look and let me know what you think. If it's ok then this PR is ready to be merged.